server_installation/topics/operating-mode/crossdc/proc-configuring-remote-cache.adoc

```diff
@@ -0,0 +1,122 @@
+
+[id="proc-configuring-remote-cache-{context}"]
+= Configuring Remote Cache Stores on {project_name}
+After you set up remote {jdgserver_name} clusters to back up {jdgserver_name} data, you can configure the Infinispan subsystem  to use those clusters as remote stores.
+
+.Prerequisites
+
+* Set up remote {jdgserver_name} clusters that can back up {jdgserver_name} data.
+* Create a truststore that contains the SSL certificate with the {jdgserver_name} Server identity.
+
+.Procedure
+
+. Add the truststore to the {project_name} deployment.
+. Create a socket binding that points to your {jdgserver_name} cluster.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<outbound-socket-binding name="remote-cache"> <1>
+  <remote-destination host="${remote.cache.host:<server_hostname>}" <2>
+                      port="${remote.cache.port:11222}"/> <3>
+</outbound-socket-binding>
+----
+<1> Names the socket binding as `remote-cache`.
+<2> Specifies one or more hostnames for the {jdgserver_name} cluster.
+<3> Defines the port of `11222` where the Hot Rod endpoint listens.
++
+. Add the `org.keycloak.keycloak-model-infinispan` module to the `keycloak` cache container in the Infinispan subsystem.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<subsystem xmlns="urn:jboss:domain:infinispan:11.0">
+    <cache-container name="keycloak" module="org.keycloak.keycloak-model-infinispan">
+----
++
+. Create a `hotrod-client.properties` file with the following content:
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+infinispan.client.hotrod.server_list = 127.0.0.1:11222
+infinispan.client.hotrod.auth_username = myuser
+infinispan.client.hotrod.auth_password = qwer1234!
+infinispan.client.hotrod.auth_realm = default
+infinispan.client.hotrod.auth_server_name = infinispan
+infinispan.client.hotrod.sasl_mechanism = DIGEST-MD5
+infinispan.client.hotrod.trust_store_file_name = /path/to/truststore.jks
+infinispan.client.hotrod.trust_store_type = JKS
+infinispan.client.hotrod.trust_store_password = password
+----
++
+. Add `hotrod-client.properties` to the classpath of your {project_name} server.
```

Comment:
This should show a specific way how to do it, e.g. via `./standalone.sh ... -P <path_to_properties_file>` parameter.